app/src/main/kotlin/com/proxerme/app/helper/MaterialDrawerHelper.kt

```diff
@@ -133,17 +158,49 @@ class MaterialDrawerHelper(context: Activity, toolbar: Toolbar,
 
     private fun buildDrawer(context: Activity, toolbar: Toolbar, accountHeader: AccountHeader,
                             savedInstanceState: Bundle?): Drawer {
-        return DrawerBuilder(context)
+        val stickyDrawerItems = generateStickyDrawerItems()
+        this.stickyItemIds = stickyDrawerItems
+                .mapNotNull { DrawerItem.fromOrNull(it.identifier) }
+                .toTypedArray()
```

Comment:
I don't really like this. How about a private util function like `stickyIds(): Array<Long>` which does this everytime? (Once here and in `onDrawerItemClick`)
This would avoid storing it (No `stickyDrawerIds` field) in this instance and it's not too expensive I think.